package.json

```diff
@@ -35,7 +35,7 @@
     "helmet": "^3.3.0",
     "method-override": "^2.3.5",
     "morgan": "^1.7.0",
-    "ngrok": "2.2.4"
+    "request": "^2.79.0"
```

Comment:
switched to using `request` module which allows use to spec the protocol in the URL.